src/components/Dropdown/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+import "./styles.scss";
+import { IDropdownProps } from "../../types/interfaces";
+import { ReactComponent as Arrow } from "../../assets/arrowDown.svg";
+export default function Dropdown({ items, selected }: IDropdownProps) {
+  const [isOpen, setIsOpen] = useState(false);
+  const handleDropdown = (event: React.MouseEvent<HTMLButtonElement>) => {
+    event.preventDefault();
+    setIsOpen(!isOpen);
```

Comment:
```suggestion
    setIsOpen(oldValue => !oldValue);
```

Reply:
What is the reason for doing it this way?